spec/helpers/pages_helper_spec.rb

```diff
@@ -27,6 +27,27 @@
     end
   end
 
+  describe 'format_ak_ui_url' do
+    it 'returns the passed value if no Settings.ak_ui_url' do
+      Settings.ak_ui_url = nil
```

Comment:
In production we already have two ak related URLs defined (both with identical values!):
`AK_HOST = https://act.sumofus.org/rest/v1` and `AK_API_URL = https://act.sumofus.org/rest/v1`.

Not sure we should be introducing a third... fancy doing some refactoring?

Really we just need `AK_DOMAIN` defined in `Settings`. `/rest/v1` and `/core/admin` are consistent across all installations of AK, and can be defined as constants.